### PR TITLE
Spanish translation completed, updated and improved

### PR DESCRIPTION
**Pull request by ibertario**
Spanish translation completed, updated and improved (completada, actualizada y mejorada la traducción).

Relevant changes (cambios relevantes):

wallet = cartera
seed = semilla
peer = contraparte
electrum = Electrum
blockchain = cadena de bloques
tarifa (fee) = comisión
Tasa de tarifa (fee rate) = Ratio de comisión

### CODE REVIEW

**Comment by mautematico** [comment]
¡muy buenos cambios!

I totally agree with:
seed
peer
electrum
blockchain
fee

I'm okay/neutral about wallet (it could be either cartera or billetera).

however, what do you think about:

fee rate => tasa de comisión
fee rate => proporción de comisión
fee rate => comisión proporcional

Any of them would be a better fit in my opinion, although last one would need some contextual changes on the strings around that.

Reason on this suggestion is that ratio is not even a Spanish word.

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -235,64 +275,74 @@
   <string name="listallchannels_share_all">Compartir todos los canales…</string>
   <string name="listallchannels_share">Compartir</string>
   <string name="listallchannels_funding_tx">Transacción</string>
-  <string name="listallchannels_funding_tx_desc">Ver transacción de pagos</string>
+  <string name="listallchannels_funding_tx_desc">Ver transacción de financiación</string>
```
Comment (mautematico) [inline app/src/main/res/values-es/strings.xml:278]: What do you think about `Ver transacción de apertura del canal` or even the shortest one : `Ver transacción de apertura`?

`Transacción de financiamiento` could also work just fine and keep the relevance of`funding`

https://www.diccionariodedudas.com/financiacion-o-financiamiento/

**Comment by ibertario** [comment]
Thanks for take time with my PR!

> I'm okay/neutral about wallet (it could be either cartera or billetera).

The reason is because in es-ES is more common "**Cartera**" and I think that in other variants it is also well understood.



> fee rate => tasa de comisión

"Tasa" y "Comisión" are usually used colloquially for the same purposes.

> fee rate => proporción de comisión

It would be a part of the fee.

> fee rate => comisión proporcional

Here "proporcional" is an adjective of "comisión", like the other option "fija" (fixed).

I find a lot of problems using this terms, which can be misunderstood, unless the context is changed or the explanation is expanded, which I do not see feasible from here. That's why my choice "**Ratio de comisión**".

> Any of them would be a better fit in my opinion, although last one would need some contextual changes on the strings around that.
> 
> Reason on this suggestion is that ratio is not even a Spanish word.

It is! Widely used in science and understood also colloquially: https://dle.rae.es/ratio

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -235,64 +275,74 @@
...
+  <string name="channel_error">No se han podido recuperar los detalles de este canal. Por favor, inténtalo de nuevo más tarde.</string>
 
   <!-- //////////////// menu //////////////// -->
 
-  <string name="menu_settings">Configuración</string>
-  <string name="menu_send">Enviar</string>
-  <string name="menu_receive">Recibir</string>
+  <string name="menu_settings">Settings</string>
+  <string name="menu_send">Send</string>
+  <string name="menu_receive">Receive</string>
```
Comment (mautematico) [inline app/src/main/res/values-es/strings.xml:293]: Oops!

Please revert this.

File: app/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -235,64 +275,74 @@
...
 
   <!-- //////////////// main //////////////////// -->
 
-  <string name="main_alert_electrum_connecting">Conectando al servidor electrum…</string>
+  <string name="main_alert_electrum_connecting">Conectando al servidor Electrum…</string>
   <string name="main_alert_network_lost">No hay conexión. Por favor verifica la configuración de tu red…</string>
-  <string name="main_alert_lightning_connection_lost">No se pudo conectar a la red Lightning…</string>
-  <string name="main_alert_settings_button">Configuración</string>
+  <string name="main_alert_lightning_connection_lost">No se ha podido conectar a la red Lightning…</string>
+  <string name="main_alert_settings_button">Settings</string>
```
Comment (mautematico) [inline app/src/main/res/values-es/strings.xml:305]: Please revert this

File: app/src/main/res/values-es/strings.xml [hunk 4]
```diff
@@ -235,64 +275,74 @@
...
-  <string name="inappnotif_mnemonics_never_seen_action">Crear copia de seguridad de mi wallet</string>
-  <string name="inappnotif_background_worker_cannot_run">Phoenix necesita poder ejecutarse en segundo plano de vez en cuando. Asegúrate de que tu dispositivo no elimine agresivamente esta aplicación para ahorrar batería.</string>
+  <string name="inappnotif_mnemonics_never_seen">Gracias por usar Phoenix. ¿Ya hiciste una copia de seguridad de tu cartera?</string>
+  <string name="inappnotif_mnemonics_never_seen_action">Crear copia de seguridad de mi cartera</string>
+  <string name="inappnotif_background_worker_cannot_run">Phoenix necesita poder ejecutarse en segundo plano de vez en cuando. Asegúrate de que tu dispositivo no fuerce el cierre de esta aplicación para ahorrar batería.</string>
   <string name="inappnotif_upgrade">Hay una actualización disponible para esta aplicación.</string>
-  <string name="inappnotif_upgrade_critical">Una actualización crítica está disponible. Tu wallet puede funcionar mal si no la actualizas.</string>
+  <string name="inappnotif_upgrade_critical">Una actualización crítica está disponible. La cartera puede funcionar mal si se actualiza.</string>
```
Comment (mautematico) [inline app/src/main/res/values-es/strings.xml:320]: You missed the word `no`.

Also consider that is the user who updates the wallet; i.e. the wallet is not updated by itself. That's the reason behing `si no la actualizas`.

File: app/src/main/res/values-es/strings.xml [hunk 5]
```diff
@@ -301,25 +351,29 @@
...
+  <string name="paymentdetails_status_sent_failed"><![CDATA[<b>Fallo</b> en el pago.<br />No se ha enviado dinero.]]></string>
+
   <string name="paymentdetails_status_received_pending">El pago aún no se ha recibido.</string>
   <string name="paymentdetails_status_received_successful"><![CDATA[<b>Recibido</b> %1$s]]></string>
+
   <string name="paymentdetails_desc_label">Desc.</string>
-  <string name="paymentdetails_destination_label">Enviar a</string>
-  <string name="paymentdetails_fees_label">Comisión</string>
+  <string name="paymentdetails_destination_label">Enviado a</string>
+  <string name="paymentdetails_fees_label">Comisiones</string>
```
Comment (mautematico) [inline app/src/main/res/values-es/strings.xml:371]: destination_label, fees_label, desc_label are being used both in the sending and the payment details screens:

image

So `Enviado a` is not a good fit for the _sending_ screen, and `Enviar a` is not a perfect fit for the _details screen_. I personally think that the sending screen is more important than the details one, so If I had to choose, I would prefer `Enviar a` over `Enviado a`.

What do you think about `Destino`? I think it is not a perfect label, but anyway a good fit for both screens.

File: app/src/main/res/values-es/strings.xml [hunk 6]
```diff
@@ -336,84 +390,158 @@
   <string name="state_wait_confirmed">La transacción de apertura del canal está siendo confirmada</string>
   <string name="state_closing">El canal se está cerrando</string>
   <string name="state_closed">El canal se ha cerrado</string>
-  <string name="state_normal">En funcionamiento</string>
-  <string name="state_sync">Sincronizando con compañero</string>
-  <string name="state_offline">El compañero está desconectado</string>
+  <string name="state_normal">En línea y funcionando</string>
+  <string name="state_sync">Sincronizando con la contraparte</string>
+  <string name="state_offline">La contraparte está desconectada</string>
```
Comment (mautematico) [inline app/src/main/res/values-es/strings.xml:395]: ¡Me gustaron mucho estos cambios que hiciste!

File: app/src/main/res/values-es/strings.xml [hunk 7]
```diff
@@ -336,84 +390,158 @@
...
-  <string name="electrum_subtitle">Por defecto, Phoenix se conecta a servidores electrum aleatorios para acceder a la blockchain de Bitcoin. También puedes optar por conectarte a tu propio servidor Electrum.</string>
+  <string name="electrum_subtitle">Por defecto, Phoenix se conecta a servidores Electrum aleatorios para acceder a la cadena de bloques de Bitcoin. También puedes optar por conectarte a tu propio servidor Electrum.</string>
   <string name="electrum_change_button">Establecer servidor…</string>
-  <string name="electrum_block_height_label">Altura del bloque</string>
+  
+  <string name="electrum_block_height_label">Altura de bloque</string>
   <string name="electrum_tip_label">Marca de tiempo</string>
-  <string name="electrum_fee_rate_label">Tasa de tarifa</string>
-  <string name="electrum_xpub_label">Clave maestra pública</string>
+  <string name="electrum_fee_rate_label">Ratio de comisión</string>
```
Comment (mautematico) [inline app/src/main/res/values-es/strings.xml:478]: Please consider

`Tasa de comisión`
`Comisión proporcional`

or some other alternative, as ratio is not even a Spanish word.

Comment (mautematico) [reply app/src/main/res/values-es/strings.xml:478]: >It is! Widely used in science and understood also colloquially: https://dle.rae.es/ratio

O M G

Comment (mautematico) [reply app/src/main/res/values-es/strings.xml:478]: I have seen a lot of "ratio de conversión" over there, but I always thought it was an anglicism.
Even now, I still think it was recently included or something. xP

I also understand the confusion about "tasa" being used in some parts as "comisión", so let's go with whatever you think it's the best choice.

File: app/src/main/res/values-es/strings.xml (hunk 3 above)
Comment (mautematico) [reply app/src/main/res/values-es/strings.xml:305]: I mean the `Settings` one.

File: app/src/main/res/values-es/strings.xml (hunk 4 above)
Comment (mautematico) [reply app/src/main/res/values-es/strings.xml:320]: > You missed the word `no`.
> 
> Also consider that is the user who updates the wallet; i.e. the wallet is not updated by itself. That's the reason behind `si no la actualizas`; however that is more like an style and not a technical reason, so go with whatever you consider the best.

**Comment by mautematico** [comment]
Dejé otros comentarios en el xml.
Faltaba traducir muchas frases al español, muchas gracias por completar la traducción. Ojalá los chicos de ACINQ fusionen pronto tus cambios.

**Comment by ibertario** [comment]
> Dejé otros comentarios en el xml.
> Faltaba traducir muchas frases al español, muchas gracias por completar la traducción. Ojalá los chicos de ACINQ fusionen pronto tus cambios.

Menuda revisión de la revisión! En cuanto pueda me pongo con todo, es increible que haya dejado algunos fallos...

Espero aclararme que es mi primer PR en GitHub, gracias!

File: app/src/main/res/values-es/strings.xml [hunk 8]
```diff
@@ -130,43 +159,50 @@
...
   <string name="scan_amountless_legacy_message"><![CDATA[
-    Esta factura no incluye una cantidad. Esto puede ser peligroso: los nodos maliciosos pueden robar tu pago. Por tu seguridad, deberías <b>pedirle al beneficiario que especifique una cantidad</b> en la solicitud de pago.
+    Esta solicitud de pago no incluye una cantidad. Esto puede ser peligroso: nodos maliciosos podrían ser capaces de robar tu pago. Por tu seguridad, deberías <b>pedir al beneficiario que especifique una cantidad</b> en la solicitud de pago.
     <br /><br />
-    ¿Quieres pagar esta solicitud de todos modos?
+    ¿Seguro que quieres pagar esta solicitud?
   ]]></string>
-  <string name="scan_amountless_legacy_confirm_button">Confirmar</string>
-  <string name="scan_amountless_legacy_cancel_button">Cancelar</string>
+  <string name="scan_amountless_legacy_confirm_button">Confirm</string>
```
Comment (ibertario) [inline app/src/main/res/values-es/strings.xml:182]: Changed.

File: app/src/main/res/values-es/strings.xml [hunk 9]
```diff
@@ -195,27 +234,28 @@
...
+    Esta característica no es un &quot;botón mágico para arreglarlo todo&quot;; está aquí como medida de seguridad y <b>solo debe usarse en escenarios extremos</b>. Por ejemplo, si tu contraparte (ACINQ) desaparece de forma permanente, impidiendo que gastes tu dinero. En cualquier otro caso, <b>si tienes problemas con Phoenix, debes contactar al equipo de soporte</b>.
     <br/><br/>
-    Forzar el cierre de canales te costará dinero (para cubrir las tarifas en la cadena de bloques) y hará que tus fondos estén bloqueados durante días. <b>No desinstales la aplicación hasta que tus canales estén completamente cerrados, o perderás tu dinero.</b>
+    Forzar el cierre de canales te costará dinero (para cubrir las comisiones de la cadena de bloques) y hará que tus fondos estén bloqueados durante días. <b>No desinstales la aplicación hasta que tus canales estén completamente cerrados, o perderás dinero.</b>
     <br/><br/>
-    <b>No uses esta función si no entiendes exactamente las consecuencias.</b>
+    <b>No uses esta función si no entiendes copmletamente como funciona.</b>
```
[the remark this answers is not in the document]
Comment (ibertario) [reply app/src/main/res/values-es/strings.xml:253]: Thanks for that! It's a common mistake and I always forget the rule. "cómo" is the correct form here.
Also, "copmletamente" is "completamente"

File: app/src/main/res/values-es/strings.xml [hunk 10]
```diff
@@ -195,27 +234,28 @@
...
   ]]></string>
   <string name="closechannels_force_destination">Los fondos eventualmente se enviarán a:</string>
+  <string name="closechannels_force_destination_desc">Esta dirección se deriva de tu semilla y te pertenece.</string>
   <string name="closechannels_force_button">Forzar el cierre de canales</string>
-  <string name="closechannels_confirm_dialog_message">¿Aún así quieres proceder?</string>
+  <string name="closechannels_confirm_dialog_message">¿Seguro que quieres proceder?</string>
```
[the remark this answers is not in the document]
Comment (ibertario) [reply app/src/main/res/values-es/strings.xml:258]: Done.

File: app/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (ibertario) [reply app/src/main/res/values-es/strings.xml:278]: "Ver transacción **de apertura del canal**" is perfect, thanks.

File: app/src/main/res/values-es/strings.xml (hunk 2 above)
Comment (ibertario) [reply app/src/main/res/values-es/strings.xml:293]: I used the original english file, that's the reason for this and other I founded, thanks for catch this!

File: app/src/main/res/values-es/strings.xml (hunk 3 above)
Comment (ibertario) [reply app/src/main/res/values-es/strings.xml:305]: Done

File: app/src/main/res/values-es/strings.xml (hunk 4 above)
Comment (ibertario) [reply app/src/main/res/values-es/strings.xml:320]: Well, in this case, I think that is the app wich is updated, so "La **aplicación** puede funcionar mal si **no** se actualiza" is more accurate and follows the previous sentence.

File: app/src/main/res/values-es/strings.xml (hunk 5 above)
Comment (ibertario) [reply app/src/main/res/values-es/strings.xml:371]: "Destino" is very good for this! Done.

File: app/src/main/res/values-es/strings.xml (hunk 7 above)
Comment (ibertario) [reply app/src/main/res/values-es/strings.xml:478]: Resolved previously!

**Comment by ibertario** [comment]
Sorry @mautematico , I reviewed before updating the translation file, this was the first time I have made changes after a review.

**Comment by ibertario** [comment]
I reviewed before updating the translation file and now here appears as "Outdated" with the previous version.

Sorry @mautematico, this was the first time I have made changes after a review.

Here are the changes: https://github.com/ibertario/phoenix/commit/aaceebf9ea4e9a4deeb6b18f55b24a698232fe32

**Comment by dpad85** [comment]
I've merged master into your PR to resolve conflicts.

Thank you @ibertario for this PR and for your patience, and thanks @mautematico for the review.

**Pull request merged**